tests/test.cpp

```diff
@@ -1346,8 +1346,15 @@ void EnumNamesTest() {
   TEST_EQ_STR("Red", EnumNameColor(Color_Red));
   TEST_EQ_STR("Green", EnumNameColor(Color_Green));
   TEST_EQ_STR("Blue", EnumNameColor(Color_Blue));
-  TEST_EQ_STR("", EnumNameColor(static_cast<Color>(-1)));
-  TEST_EQ_STR("", EnumNameColor(static_cast<Color>(1000)));
+  // Check that Color to string don't crash while decode a mixture of Colors.
```

Comment:
@fbenkstein
Could you review this code?
Are you agree with these changes and comments?

Reply:
Looks good to me.